docs/docs/tutorial/chapter3/saving-data.md

```diff
@@ -1280,7 +1281,7 @@ export const createContact = ({ input }) => {
 <TabItem value="ts" label="TypeScript">
 
 ```ts title="api/src/services/contacts/contacts.ts"
-import type { Prisma } from '@prisma/client'
+import type { QueryResolvers, MutationResolvers } from 'types/graphql'
```

Comment:
I think the type for createContact (line 1291) needs to be updated too no?

Reply:
Oh! Yes, nice catch